src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,26 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
+    Set<Employee> employeeSet;
```

Comment:
why have you moved this variable to the class level?

Reply:
automatically, in many recent tasks first we were declaring fields.

Reply:
moved to local variable in method